bridge/client/app/_components/ktb-sli-breakdown/ktb-sli-breakdown.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { KtbSliBreakdownComponent } from './ktb-sli-breakdown.component';
+import { KtbSliBreakdownCriteriaItemComponent } from './ktb-sli-breakdown-criteria-item.component';
+import { DtTableModule } from '@dynatrace/barista-components/table';
+import { DtIconModule } from '@dynatrace/barista-components/icon';
+import { KtbPipeModule } from '../../_pipes/ktb-pipe.module';
+import { FlexModule } from '@angular/flex-layout';
+import { DtOverlayModule } from '@dynatrace/barista-components/overlay';
+
+@NgModule({
+  declarations: [KtbSliBreakdownComponent, KtbSliBreakdownCriteriaItemComponent],
+  imports: [
+    CommonModule,
+    DtTableModule,
+    DtIconModule.forRoot({
+      svgIconLocation: `assets/icons/{{name}}.svg`,
+    }),
+    KtbPipeModule,
+    FlexModule,
+    DtOverlayModule,
+  ],
+  exports: [KtbSliBreakdownComponent, KtbSliBreakdownCriteriaItemComponent],
```

Comment:
Since `KtbSliBreakdownCriteriaItemComponent` is only used by `KtbSliBreakdownComponent`, we could also omit it from the exports, what do you think?